packages/app-extensions/src/appFactory/App.js

```diff
@@ -45,7 +43,8 @@ App.propTypes = {
   store: PropTypes.object.isRequired,
   initIntlPromise: PropTypes.object.isRequired,
   content: PropTypes.node.isRequired,
-  theme: PropTypes.object
+  theme: PropTypes.object,
+  themeType: PropTypes.string
```

Comment:
I would prefer here explicit prop types for better understanding:
`PropTypes.oneOf(['WIDGET', 'ADMIN'])`